\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{swThesis}
\usepackage{amsmath}
\usepackage{lipsum}
\usepackage{standalone}
\usepackage{epigraph}  % For dedication and quote
\usepackage{bibentry}  % Allow full citations in text
\standalonetrue

\bibliography{bibliography}

\begin{document}

\chapter*{Declaration}
\addcontentsline{toc}{chapter}{Declaration}
\setlength{\epigraphrule}{0pt}
\setlength{\epigraphwidth}{\textwidth}
\epigraph{\normalsize
This thesis presents my own work, and has not been submitted for any other degree or professional qualification.
Wherever results were obtained in collaboration with others, I have clearly stated it in the text.
Any information derived from the published work of others has been cited in the text, and a complete list of references can be found in the bibliography.
Published papers arising from the work described in this thesis can be found in the appendices.}
{\normalsize \vspace{0.8cm} -- Scott Warchal, 2018}

% Dedication

\chapter*{}
\setlength{\epigraphwidth}{0.4\textwidth}
\epigraph{\textit{epigraph here}}

\chapter*{Acknowledgements}
\addcontentsline{toc}{chapter}{Acknowledgements}
Acknowledgements here.

% 1 page max
\chapter*{Abstract}
\addcontentsline{toc}{chapter}{Abstract}
High content image-based screening assays utilise cell based models to extract an quantify morphological phenotypes induced by small molecules.
The rich datasets produced can be used to identify lead compounds in drug discovery efforts, infer compound mechanism of action, or aid biological understanding with the use of tool compounds.
Here I present my work developing and applying high-content image based screens of small molecules across a panel of eight genetically and morphologically distinct breast cancer cell lines.

I implemented machine learning models to predict compound mechanism of action from morphological data and assesed how well these models transfer to unseen cell lines, comparing the use of numeric morphological features extracted using computer vision techniques against more modern convolutional neural networks acting on raw image data.

The use of cell lines panels have been widely used in pharmacogenomics in order to compare the sensitivity between genetically distinct cell lines to drug treatments.
I applied dimensional reduction techniques and distance metrics to develop a measure of differential morphological response between cell lines to small molecule treatment, which controls for the inherent morphological differences between untreated cell lines.

These methods were then applied to a screen of 13,000 drug-like molecules across the eight cell lines to identify compounds which produced distinct phenotypic responses between cell lines.
Putatative hits were then validated in a three-dimensional tumour spheroid assay to determine the functional effect of these compounds in more complex models, as well as proteomics to determine the responsible pathways.

Using data generated from the compound screen, I carried out work towards integrating knowledge of chemical structures with morphological data to infer mechanistic information of the unannotated compounds, and assess structure activity relationships from imaging data.


\chapter*{Lay Summary}
\addcontentsline{toc}{chapter}{Lay summary}
Lay summary here.

\clearpage
\tableofcontents
\addcontentsline{toc}{chapter}{Contents}

\clearpage
\listoffigures
\addcontentsline{toc}{chapter}{List of Figures}

\clearpage
\listoftables
\addcontentsline{toc}{chapter}{List of Tables}
\clearpage

\chapter*{List of Acronyms}
\addcontentsline{toc}{chapter}{List of Acronyms}

\begin{acronym}
    \acro{2D}{Two-dimensional}
    \acro{3D}{Three-dimensional}
    \acro{ABL}{Abelson murine leukemia viral oncogene homologue}
    \acro{BCR}{Breakpoint cluster region}
    \acro{ANN}{Artificial neural network}
    \acro{BSA}{Bovine serum albumin}
    \acro{CCLE}{Cancer cell line encyclopedia}
    \acro{CCM}{Cerebral cavernous malformation}
    \acro{CNN}{Convolutional neural networks}
    \acro{DMEM} Dulbecco's modified eagle medium
    \acro{DMSO}{Dimethyl sulfoxide}
    \acro{EMA}{European Medicines Agency}
    \acro{FDA}{U.S Food and Drug Administration}
    \acro{GPU}{Graphics processing unit}
    \acro{HTS}{High throughput screening}
    \acro{MCL}{Markov clustering algorithm}
    \acro{MOA}{Mechanism of action}
    \acro{mRMR}{Minimum-redundancy-maximum-relevancy}
    \acro{PBS}{Phosphate buffered saline}
    \acro{PCA}{Principal component analysis}
    \acro{PDD}{Phenotypic drug discovery}
    \acro{RGB}{red green blue}
    \acro{SAR}{Structure activity relationship}
    \acro{TCCS}{Theta comparative cell scoring}
\end{acronym}

\end{document}
