\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{swThesis}
\usepackage{lipsum}
\usepackage{standalone}
\usepackage{caption}
\standalonetrue

% Bibliography
\bibliography{bibliography}

% Figures
\graphicspath{./figs}

\begin{document}

\chapter{Introduction} \label{chapter:intro}

\section{Eroom's Law: The increasing cost of drug discovery}
Throughout the last 70 years the cost of developing a new drug has steadily increased.
Scannel \textit{et al.} observed that the cost to develop a new drug has approximately doubled every 9 years \cite{Scannell2012}.
This observation has been dubbed ``Eroom's law'', a homage to Moore's law -- a well-known observation that the number of transistors in microprocessors approximately doubles every 2 years.
The cost of bringing a new drug to market is now approaching Â£1 billion, taking 10 years from initial concept to approval, the reasons behind this every-increasing cost are multi-faceted.
One explanation may be that the low-hanging fruit has been taken, effective long-standing remedies have been studied and commercialised, natural products screened, and we are now tackling the more complex diseases and pharmacological targets.
% pharma's productivity cliff
% lots of interest in diverse approaches to drug-discovery
% incorporate more technology intro drug-discovery

\section{The drug discovery process}
\texttt{TODO: flesh this out}
Commercial drug discovery programmes usually follow a well established pattern:
\begin{enumerate}
    \item For a given disease or indication, an assay is developed which allows for high-throughput screening of large chemical libraries.
    \item Hits are selected based on a univariate read-out. These are then confirmed with further repeats.
    \item Lead compounds are then validated by testing in more complex assays, such as \textit{in vitro} or \textit{in vivo} disease models.
    \item Validated compounds are taken through DMPK and toxicity testing.
    \item A lead compound is selected and taken onto clinical trials.
        \begin{enumerate}
            \item phase I: Toxicity and DMPK testing.
            \item phase II: Effectiveness at treating the condition.
            \item phase III: Larger scale testing of efficacy.
        \end{enumerate}
\end{enumerate}
\texttt{why is this being written about??}


\subsection{Target-based screening}
Over the past 30 years the majority of drug discovery programmes seized upon new advances in robotics and automation to screen ever expanding compound libraries against pre-defined protein targets.
It would be difficult to argue that this target-based high-throughput screening (HTS) approach has not been fruitful, as it has yielded many succesful therapeutics.
Despite numerous clinical and commerical success stories, HTS is not a panacea, with a high attrition rate of lead compounds once they enter clincial trials.
A large majority of these clinical trial failures are not due to toxicity, but rather a lack of efficacy, which can often be traced back to a poorly hypothesised target in the face of complex disease aetiology.

% issues with prior identification of a target
% large gamble on the hypothesised target
% examples where this has failed - Alzheimers? neuroscience in general?

\subsection{Phenotypic screening}
Phenotypic screening differs from target-based screening in that it does not rely on prior knowledge of a specific target, but instead relies on interrogating a biologically relevant assay to identify hit compounds.
This target-agnostic approach can prove useful in diseases with poorly understood mechanisms, or those with no obvious druggable protein targets.

One such example is the anti-hepatitis C drug daclatasvir, which was discovered through a phenotypic assay in which human cells were modified to express the hepatitis C virus (HCV) replicon.
This cell-model was then screened to identify compounds which reduced HCV replication.
The target for daclatasvir was later found to be the HCV NS5A protein, which would not have been suggested in a target-based screen, as at the time it was thought to have no involvement with HVC.
This elegant assay has the very useful attribute of modelling the disease, whilst simulatenously weeding out many compounds that may prove to be toxic in human cells. % TODO: citation for daclatasivir


% define what is is
% how is this different to target-based
% historic approach
% examples of approved drugs through phenotypic screening (monastrol?)
% resurgence in recent years -> swinney and anthony, first in class
% pros, cons (complimentary to target-based, not a replacement)

\section{High content imaging}
High content imaging is often thought of as a sub set of phenotypic screening.
% how this relates to phenotypic screening
% how this can be used in drug-discovery
% benefits over transcriptomics, proteomics etc
% downsides compared to alternatives

\section{Cancer}
% general background on breast cancer
% rationale behind picking the 8 cell lines
% cancer drug discovery
% applications of phenotypic/high-content imaging in cancer

% Please add the following required packages to your document preamble:
% \usepackage{booktabs}
\begin{table}[]
    \captionsetup{width=0.70\textwidth}
    \centering
    \caption[Panel of breast cancer cell lines chosen for study]{Panel of breast cancer cell lines chosen for study. PI3K:Phosphoinsitide-3-kinase, PTEN:Phosphatase and tensin homolog, ER:Estrogen receptor, TN:triple-negative, HER2:human epidermal growth factor, WT:wild-type, *:lack of consensus regarding the mutational status}
    \label{table:cell-lines}
    \begin{tabular}{@{}llll@{}}
    \toprule
               &                    & \multicolumn{2}{l}{Mutational status} \\
    Cell line  & Molecular subclass & PTEN             & PI3K               \\ \midrule
    MCF7       & ER                 & WT               & E545K              \\
    T47D       & ER                 & WT               & H1047R             \\
    MDA-MB-231 & TN                 & WT               & WT                 \\
    MDA-MB-157 & TN                 & WT               & WT                 \\
    HCC1569    & HER2               & WT               & WT                 \\
    SKBR3      & HER2               & WT               & WT                 \\
    HCC1954    & HER2               & *                & H1047R             \\
    KPL4       & HER2               & *                & H1047R             \\ \bottomrule
    \end{tabular}
    \end{table}


%\printbibliography

\end{document}
