\documentclass[a4paper,11pt,twoside,openright]{scrbook}

\usepackage{swThesis}
\usepackage{amsmath}
\usepackage[htt]{hyphenat}
\usepackage{lipsum}
\usepackage{standalone}
\standalonetrue

\bibliography{bibliography}

% Figures
\graphicspath{./figs}

\begin{document}

\chapter{Cheminformatics and high-content imaging} \label{chapter:cheminformatics}

\section{Introduction}

\subsection{Cheminformatics}

The term ``cheminformatics'' was first coined in 1998\cite{Brown1998} although the use of computers to interact with 
chemical data predates this by many years with early systems used to index, search and catalogue databases of chemical 
compounds. \cite{Ray1957}
Most of the early work in this field was concerned with efficient means to search chemical databases for similar 
molecules or molecules containing certain sub-structures.
This early work developed a number of important methods to generate, represent and compare chemical structures in a 
time of limited computational power, as a by-product these methods are very efficient and are still used today as the 
size of chemical databases has grown alongside computational power.

It was later on that researchers attempted to correlate biological activity and physiochemical parameters with 
structure activity relationships (SAR), this was partly due to the advancement of statistical techniques which gave 
rise to new tools such as multiple linear regression.
One of the first quantitative SAR (QSAR) studies was carried out by Hansch and Fujita, in which they found the 
lipophilicity of a molecule correlated strongly with biological activity. \cite{Hansch1962}
Since then the QSAR field has advanced to include many more parameters and in now a key part in most empirical drug 
discovery efforts.

Another use of cheminformatics in drug discovery is the analysis and design of compound screening libraries.
In industrial high-throughput screening a full-deck compound library typically contains several million small 
molecules, screening this entire library is a costly endeavour, even for pharmaceutical companies, and therefore a lot 
of research has been carried out in how to maximise the value and information gained from screening large compound 
collections.
One of the ways compound libraries can be optimised is by covering a large a range of chemical space as possible.
A compound library that contains many extremely similar molecules may be useful in certain specific circumstances, but 
in most cases this is viewed as a redundancy and a library which covers the same chemical space with fewer compounds 
would reduce costs.
Alternatively, a compound collection of equal size which contains more diverse chemistry may lead to a more varied 
selection of lead candidates. \cite{Clemons2011}
The concept of chemical space in compound collections can also be used to identify potential blind-spots or bias in 
drug discovery libraries, which are areas of chemical space with potential biological potential that are not covered by 
an existing library, in contrast to areas of chemical space which are well covered by a compound collection but have 
historically failed to show biological activity, termed ``dark chemical matter''. \cite{Wassermann2015}


\subsection{Structure activity relationships}

A structure activity relationship is the link between a chemical's structure and its effect in a biological system, 
which underpins much of the medicinal chemistry field.
The underlying premise of SAR is that compounds with similar structures and physiochemical properties have similar 
biological effects by virtue of binding to the same or similar targets.
This idea is commonly applied during lead optimisation whereby a candidate molecule is iteratively modified in order to 
optimise parameters such as specificity and affinity, all the while ensuring that these modifications do not disrupt 
binding to the desired target, leading to the identification and determination of functional groups which are required 
for target engagement and biological activity.

Relating changes to a compound's structure to biological activity is relatively straightforward if compound activity 
can be represented as a single variable such as binding affinity or EC$_{50}$, applying quantitative SAR (QSAR) to 
multiparametric data such as that found with high-content imaging is not as well defined.




\subsection{Chemical similarity}

The premise of QSAR is ``\textsl{similar molecules} have similar biological effects'' presenting the challenge of how 
to measure similarity between chemical structures.
Chemical structures can be represented in a number of different formats and we typically think of the skeletal 2D 
graphical representation (figure \ref{figure:mol_encodings} A) when considering complex organic molecules which have to 
be interpreted by chemists.
Computers however require a different format to efficiently store and parse chemical structure data.
SMILEs (simplified molecular input line entry system) and InChIs (international chemical identifier) are two formats 
which encode chemical structures as short character strings representing atoms as human readable characters (such as 
\texttt{CH} for carbon and hydrogen) with other symbols to represent branches and stereochemistry (figure 
\ref{figure:mol_encodings} B\&C).
These relatively simple formats sometimes suffer from ambiguity, in which a single encoding could represent several 
molecules, or a single molecule could be
represented by multiple valid encodings.
A less ambiguous but also less human-readable file format is SDF (structure data file) or Molfile, which encode 
chemical structures as a table of x, y, z co-ordinates and bonds for each atom (figure \ref{figure:mol_encodings} D).

\begin{figure}
    \fcapsideright{
    \caption[Different methods to encode chemical structure]{
Different methods to encode the chemical structure of a molecule (aspirin).
\textbf{(A)} A 2D skeletal graphical representation commonly used by chemists.
\textbf{(B)} SMILE format, a concise relatively human readable format encoding atoms as characters.
\textbf{(C)} InChI format, another commonly used string format which is less human readable but contains more details 
to reduce ambiguity.
\textbf{(D)} SDF / Mol format. A tabular format which lists the co-ordinates of atoms in 3 dimensions along with bonds 
and distances.
}
} {
    \includegraphics[scale=1.5]{ch5molEncodings}
    \label{figure:mol_encodings}
}
\end{figure}

Given these encodings of chemical structure and the task to calculate similarity (or distance) between molecules, the 
most direct and simple method is to calculate distance based on the string encodings (usually SMILE format), such as 
hamming distance or longest-common-substring divided by total length between two SMILE strings.\cite{Ozturk2016}
However, these naive methods  suffer from a number of drawbacks, mainly stemming from the ambiguity and variability of 
SMILE encodings which limit their widespread use in chemical similarity calculations.
A more nuanced approach to measuring chemical similarity is to first calculate  compound fingerprints such as daylight 
or extended connectivity fingerprints (ECFP) \cite{Rogers2010} which are abstract representations of molecules in the 
form of fixed-length binary arrays -- generated from local patterns in the molecule such as the identity of 
neighbouring atoms (where neighbouring is extended to several bonds away).
The distance between compound fingerprints can then be found using one of a variety of distance metrics.
To compare the binary compound fingerprints the most commonly used metric is Tanimoto similarity ($T_s$) and 
distance\footnote{Whilst not a distance in the  strict mathematical sense it is commonly referred to as a distance 
metric.} ($T_d$), where $T_s$ is defined as the ratio of common elements between two equal length fingerprints divided 
by the length of either fingerprint, and $T_d = -\log_2(T_s)$.
Another approach to molecular fingerprinting is to summarise the 3D shape of a molecule.
Ultrafast shape recognition (USR) was developed and used for \textit{in silico} drug screening to efficiently describe 
molecular shape in 12 measurements.
USR however is optimised for computational efficiency at the expense of detailed information and is agnostic to the 
atom types contained in the molecule.
This drawback led to an extension of USR (USRCAT - USR with CREDO atom types) which was later developed for users to 
search the protein data bank and describes a molecule's 3D shape and constituent atoms. \cite{Schreyer2012}
Recently a number of studies have leveraged advances in the machine learning field to generate alternative chemical 
fingerprints using neural networks. \cite{Kearnes2016a,Feinberg2018,Ma2018,Liu2018}
The idea behind these methods is that deep neural networks are able to learn appropriate representations of the input 
data in order to maximise performance in a certain task.
They typically represent chemical structures as un-directional graphs of atoms, and apply convolutional techniques -- 
which have proven themselves in image-related tasks -- to the graph structures to generate molecular fingerprints which 
can be used in downstream machine learning and cheminformatics work.


\subsection{Application of cheminformatics to high-content screening}
Much of the work in cheminformatics is carried out in industrial rather than academic laboratories, coupled with the 
relatively immature field of high-content imaging has resulted in a sparsity of published research in the application 
of cheminformatics to high-content imaging and screening.

One of the earliest papers which combined cheminformatics with image-based screening was by Young \textit{et 
al.}\cite{Young2008} who screened a library of 6,547 compounds in HeLa cells and extracted 30 morphological features 
regarding nuclei morphology.
They used factor analysis and hierarchical clustering to group their compound library into 7 clusters describing 
similar nuclear morphologies, and created matrices of phenotypic similarity with cosine similarity of phenotypic 
features and compound similarities with Tanimoto coefficients of ECFP features.
They then found a correlation between the rank ordering of phenotypic similarities and compound similarities, as well 
as identified instances of ``activity-cliffs'' when two structurally similar compounds demonstrated very different 
phenotypic activities which matched up to known SAR studies on the two compounds.

A second study by Wawer \textit{et al.}\cite{Wawer2014b} incorporated high-content morphological profiling to construct 
compound libraries based on the diversity of biological response as opposed to diversity of chemical space.
Using a library of 31,000 compounds, they performed a image-based screen and selected a subset of compounds which 
produced a diverse range of bioactivities defined with cell morphology.
They then compared this subset to a second subset generated by maximising diversity of chemical space, and investigated 
the performance of each subset of compounds in a wide range of previously performed cell-based screens.
They found that subsetting compounds based on morphological diversity resulted in an increased performance compared to 
compounds chosen on chemical diversity or compounds chosen at random.

Another study published by the same group developed a method for SAR with high-dimensional profiling data, assessing 
both high-content imaging and gene expression profiling datasets.
They used pattern mining techniques originally developed in advertising and marketing to find frequently linked 
sub-structures with certain biological activities. \cite{Wawer2014c}



\subsection{The BioAscent library}
The BioAscent compound library consists of a 12,000 compound subset of a larger 125,000 chemical diversity library.
The library was designed to include compounds with drug-like properties such as adherence to Lipinksi's rule of 5 and 
avoiding known pan-assay interference compounds (PAINs).
The bioascent collection has been found to contain a considerable proportion of molecules which are likely to be 
kinase-interacting (27\%) and GPCR-interacting (20\%) according to computational models of chemical structure performed 
by the vendor.


\subsection{Aim of this chapter}
This chapter is based on work using the BioAscent compound library which is supplied with detailed structural 
information of each of the 12,000 compounds.
My aim was to incorporate this chemical information with existing public datasets and my own high-content imaging data 
in a way to aid target convolution as well as investigate the link between chemical structure structure activity 
relationship (SAR) applied to cellular morphology as an indicator of compound activity.




\section{Results}

\subsection{The BioAscent library contains clusters of phenotypically similar compounds}

In order to compare the phenotypic profiles produced by compounds in the BioAscent library, active compounds were 
selected based on on the $l_1$ norm distance from the negative control centroid (figure \ref{figure:compound_activity}).
As many of the compounds were cytotoxic and produced images containing only a few cells which do not produce robust 
morphological measurements, an activity window was used to exclude cytotoxic compounds.


\begin{figure}
    \captionsetup{width=0.8\textwidth}
    \caption[Selecting active compounds based on distance]{
Selection of active BioAscent compounds based on the $l_1$ norm distance from the DMSO negative control centroid in PCA 
space.
Lower and upper bounds of the selected compounds are indicated by dashed lines. In total 1244 compounds were selected.}
    \includegraphics[width=0.6\textwidth]{ch5compoundActivities}
    \label{figure:compound_activity}
\end{figure}

Hierarchical clustering of morphological profiles produced by these phenotypically active compounds showed that despite 
the chemical diversity of the BioAscent library, the active compounds formed distinct clusters of compounds which 
produced similar cellular morphologies (figure \ref{figure:morph_cluster} A).
To confirm the validity of the clustering, the hierarchical labels were compared with clusters found in an unsupervised 
algorithm.
The morphological profiles were embedded into 2-dimensional space using the t-SNE algorithm \cite{Maaten2008} which 
aims to preserve local structure within the data and reveals clusters of similar points in an unsupervised manner.
When these points were coloured by the cluster labels identified by hierarchical clustering they appeared to match up 
with the t-SNE embedding (figure \ref{figure:morph_cluster} B).

\begin{figure}
    \captionsetup{width=0.8\textwidth}
    \caption[Morphological clustering of the BioAscent library]{
Morphological clustering of active compounds within the BioAscent library.
    \textbf{(A)} Hierarchical clustering of the 1244 active BioAscent compounds based on a distance matrix of principal 
components.
    Clusters formed by cutting the produced dendrogram.
    \textbf{(B)} Unsupervised t-SNE clustering of active BioAscent compounds based on principal components of 
morphological features.
    Points are colour coded with cluster labels derived from the hierarchical clustering.
}
    \includegraphics[width=0.9\textwidth]{ch5phenotypicClustersBoth}
    \label{figure:morph_cluster}
\end{figure}


\subsection{The BioAscent library is chemically diverse}
The BioAscent library is marketed as chemically diverse, yet I still wanted see to what extent and if there are 
clusters of chemically similar compounds such as those based around a common scaffold.
All 12,000 BioAscent compounds were converted into molecular fingerprints to produce a Tanimoto distance matrix between 
all pairs of compound fingerprints, this was then clustered using agglomerative hierarchical clustering.
As could be predicted, the heatmaps and dendrograms did not reveal any large clusters of structurally similar compounds 
in the 12,000 compound library.
This chemical diversity continued when the compounds were filtered to only contain the phenotypically active molecules.
The use of more novel compound fingerprinting techniques such as USRCAT \cite{Schreyer2012} and autoencoded features 
\cite{Gomez-Bombarelli2016} did not increase the degree of clustering.

Rather than looking at large-scale clustering of many thousands of compounds with hierarchical clustering, I tried the 
Butina clustering method to identify small collections of structurally similar compounds.
This method does not return similarity measures, but rather groups compounds into bins of similar compounds. 
\cite{Butina1999}
After removing clusters which contained fewer than 3 compounds, this left 96 clusters, with the largest cluster 
containing 20 compounds and 58\% of the clusters containing only 3 compounds (figure \ref{figure:butina_clusters}).

\begin{figure}
    \captionsetup{width=0.8\textwidth}
    \caption[Histogram of structural cluster sizes and example of molecules within a cluster]{
        \textbf{(A)} Histogram of number of compounds within structurally similar clusters, with most clusters only 
containing 3 molecules.
        \textbf{(B)} An example of one of the structurally similar clusters as found with the Butina clustering 
algorithm. 
}
    \includegraphics[width=0.5\textwidth]{ch5butinaClusterBoth}
    \label{figure:butina_clusters}
\end{figure}



\subsection{There is little evidence that structurally similar molecules produce similar cellular morphologies}

Following the premise of SAR, structurally similar molecules are likely to share a common target, therefore activating 
the same or similar signalling pathways and producing similar cellular morphologies.
I investigated to what extent structurally similar molecules in the BioAscent library produce similar cellular 
morphologies, and also how structurally similar are compounds which were shown to produce similar phenotypes.
Using the phenotypic clusters as defined in fig.\ref{figure:morph_cluster}, I compared the structural similarity 
between compounds within these phenotypic clusters compared to a null distribution of pairs of compounds picked at 
random.
I found that compounds within phenotypic clusters were very slightly more structurally similar than compounds in the 
null distribution (figure \ref{figure:compound_pheno_corr} A, $p=1.81\times10^{-15}$, $D=0.011$, 2-sample 
Kolmogorov-Smirnov test).
In addition, I approach the problem from the opposite direction and investigated the phenotypic similarity within 
clusters of structurally similar molecules as found with the Butina clustering algorithm, compared to the phenotypic 
similarity between compounds picked at random from the pooled compound list of those contained within Butina clusters.
I again found that structurally similar molecules are more likely to produce similar cellular morphologies than 
compounds picked at random (figure \ref{figure:compound_pheno_corr} B, $p=0.037$, $D=0.018$, 2-sample 
Kolmogorov-Smirnov test).

Another approach is to see how well the distance matrix of phenotypic profiles correlates with the distance matrix of 
chemical structures.
Using Mantel's test of correlation between two distance matrices, \cite{Mantel1967} I found no significant correlation 
between the phenotypic and structural distance matrices for the active 1244 compound subset ($r = 0.02$, $p = 0.116$).


\begin{figure}
    \captionsetup{width=0.8\textwidth}
    \caption[Comparison of structural and phenotypically similar compounds]{
        \textbf{(A)} Tanimoto distance between compounds from within phenotypic clusters (as found in fig. 
\ref{figure:morph_cluster}) and between randomly paired active compounds.
        ($p=1.81\times10^{-15}$, $D=0.011$, 2-sample Kolmogorov-Smirnov test)
    \textbf{(B)} Phenotypic distance between compounds from within structurally similar clusters and between randomly 
paired phenotypic profiles.
    ($p=0.037$, $D=0.018$, 2-sample Kolmogorov-Smirnov test)
}
    \includegraphics[width=1.0\textwidth]{ch5compoundSimilarity}
    \label{figure:compound_pheno_corr}
\end{figure}





\subsection{Identifying the putative MoA of phenotypic hits with ChEMBL structure queries}

Another way to utilise the chemical structure data available with the BioAscent library is through querying publicly 
available databases such as ChEMBL for exact compounds matches or structurally similar compounds.
This returns large amounts of data from a variety of assays in which the compound or a structural analogue was screened 
against a number of targets with information relating to EC/IC$_{50}$ values, binding affinities etc.
I investigated if this historical dataset could be used to suggest putative MoAs of hits from target agnostic 
phenotypic screening assays.

For this I used the compounds within the 10 phenotypic clusters (figure \ref{figure:morph_cluster}), and for each 
cluster queried ChEMBL based on a structure similarity search to identify records for either the query compound, or 
structural analogues.
Then using these compounds identifying which human proteins they have been screened against, and filtering these 
protein based on EC/IC$_{50}$ values.
This returns a list of Uniprot accession codes which were used with interpro \cite{Finn2017} to test for enrichment of 
protein regions compared to a background.

Eight out of the ten phenotypic clusters returned at least one significantly enriched target with fold-enrichment 
ranging between 1.5 and 10.
The most significantly enriched target in 6/8 of the clusters was related to protein kinases, whereas the remaining two 
were rhodopsin-like GPCRs and adrenergic receptors.

\begin{figure}
    \captionsetup{width=0.8\textwidth}
    \caption[Interpro target enrichment]{
        Enriched interpro targets found within a phenotypic cluster of the BioAscent library when compared to a 
background of all active BioAscent compounds.
}
    \includegraphics[width=0.8\textwidth]{ch5interpro.png}
    \label{figure:interpro}
\end{figure}



\subsection{Using phenotypic screening to find ``dark chemical matter''}

An area of interest in drug discovery is finding new pharmacologically active compounds which occupy new areas of 
chemical space. \cite{Wassermann2015}
One way to incorporate the phenotypically active hits from the BioAscent library is to query historical screening 
databases by structural similarity.
To do this I took the list of 1244 phenotypically active BioAscent compounds and performed a structural similarity 
search on the ChEMBL database to look for those BioAscent compounds which have a large Tanimoto distance from all 
compounds deposited in the database.

From the 1244 active BioAscent compounds 59 (4.7\%) were found to have no structurally similar analogues in the ChEMBL 
database (figure \ref{figure:dark_mols}).
To assess if these 59 compounds contained undesirable physiochemical properties which would limit their inclusion in 
screening libraries and explain their absence from historic screening databases I used a quantitative estimate of 
drug-likeness (QED), \cite{Bickerton2012} to compare the 59 compounds from `dark chemical space' to the 1244 active 
BioAscent compounds.
The QED metric did not reveal any significant differences in desirable physiochemical properties between the two groups 
(QED$_{\text{dark compounds}} = 0.57$,  QED$_{\text{all active}} = 0.60$, 2 sample t-test $t=0.85$, $p=0.39$).


\begin{figure}
    \captionsetup{width=1.0\textwidth}
    \caption[BioAscent hits from dark chemical space]{
59 phenotypically active BioAscent compounds with no close structural analogues in the ChEMBL database.
}
    \includegraphics[width=1.0\textwidth]{ch5darkmols}
    \label{figure:dark_mols}
\end{figure}




\section{Discussion}

Cheminformatics as a field is largely overshadowed by bioinformatics in terms of academic interests and publications 
(figure \ref{figure:informatics_ngram}), it has however arguably had a greater positive impact on the design and 
identification of new small molecule therapeutics.
As high-content screening becomes more and more prevalent in drug discovery incorporation of the fields will become 
more increasingly likely.
I therefore aimed to investigate methods in which cheminformatics analyses can aid high-content and phenotypic 
screening, and also the other way round: how high-content screening and morphological profiling can inform 
cheminformatics.


\begin{figure}
    \captionsetup{width=0.8\textwidth}
    \caption[Popularity of terms `bioinformatics' and `cheminformatics' in the literature]{
Popularity of the terms `bioinformatics' and `cheminformatics' in the published literature as found using Google's 
Ngram viewer between 1980 and 2010.
y-axis represents the cumulative percentage of literature containing the term, x-axis represents the year.
}
    \includegraphics[scale=0.5]{ch5googleNgram}
    \label{figure:informatics_ngram}
\end{figure}

From the global analysis of the BioAscent compound library I failed to find any evidence of clusters of structural 
similarity, which is not surprising when using a compound library specifically designed to maximise structural 
diversity.
I did however find smaller regions of the BioAscent library consisting of a handful of structurally similar compounds 
using the Butina clustering algorithm.
The choice of using the BioAscent compound library -- rather than one of many other alternatives -- was made by what 
was available to me at the time, as large compound collections are a precious resource in academia.
In hindsight, a chemical diversity library may not have been the ideal compound collection to use for a study relying 
heavily chemical similarity measures, and a compound library which consists of clusters of structurally similar 
molecules may have resulted in different conclusions regarding chemical similarity and phenotypic similarity.

My hypothesis that structurally similar compounds should produce similar morphological changes, and therefore compounds 
that cause similar phenotypes should be structurally similar on average did not yield particularly striking results.
While I found compounds within phenotypic clusters had lower Tanimoto distances than compounds paired at random, and 
the opposite: that compounds within structurally similar clusters as found with the Butina algorithm were more 
phenotypically similar (figure \ref{figure:compound_pheno_corr}), despite statistical significance the effect size was 
small, in globally assessing correlation between the two distance matrices showed no significant correlation.
This result is largely in agreement with that of Young \textit{et al.} who found a ``modest'' correlation of 0.0074 
between between rank-ordered pairs of compounds for phenotypic similarity and structural similarity. \cite{Young2008}
One possible explanation for these low effect sizes could be due to largely uncorrelated data with small regions of 
high correlation.
I feel that a more fine-grained analysis with a carefully constructed compound collection would be better suited for 
this task, and could result in stronger evidence for the association between chemical structure and phenotype.
Another consideration to explain the largely uncorrelated data are activity-cliffs -- where a small change to a 
molecule's structure can result in large differences in biological activity.
There is no doubt that a small change to overall chemical structure can inhibit biding of a small molecule to a target 
receptor, al thought this brings into question the usefulness of chemical similarity measures, and if many of these 
activity-cliffs are artefacts caused by poorly measured `similar' compounds, which we may see change as more nuanced 
chemical similarity measures are developed.

The availability of large public datasets which can be queried with chemical identifiers such as SMILE strings is a 
great resource with a number of potential applications.
The ChEMBL database contains information for 2.2 million compounds, and the results from over a million assays and 
12,000 targets.
In my efforts to incorporate this rich dataset with the results of the high-content screen, I encountered issues 
associated with a dataset constructed from many heterogeneous sources, such as lack of information describing the 
assay, and no consistent system to label the type of assay to allow filtering of less relevant assay types.
The idea was to find existing data from assays which used the 12,000 compound BioAscent library, however none of the 
data sources used the exact BioAscent compound library, but rather there were compounds within the BioAscent library 
that are shared in other compound collections, and so the data returned by exactly matching the BioAscent compounds was 
too sparse for further analysis.
I therefore relaxed the searching criteria, and searched instead for compounds with a Tanimoto similarity greater than 
0.9 which resulted in an adequate number of results but added an additional layer of assumptions.
The enriched protein sequences found for the compounds (or similar compounds) in each phenotypic cluster consisted 
predominantly of protein kinase regions (see figure \ref{figure:interpro} for an example of one cluster).
While this did serve as a nice sanity check, in that 20\% of the BioAscent compounds are predicted to be 
kinase-interacting, it was not particularly interesting for hypothesis generation.
In addition I would warn against putting too much faith in the hypothesised protein targets: the protein targets were 
filtered using single concentration regardless of the assay type.
It is easy to envisage that a concentration which is selective to a particular protein in a cell-based assay would not 
be stringent enough when used as a cutoff in an \textit{in vitro} protein binding assay.
Another source of uncertainty is the use of tools such as DAVID and interpro to predict enriched protein regions, these 
rely on heuristics and combining another set of heterogeneous datasets which in turn have their own errors and biases.

The concept of dark chemical matter was introduced by Wasserman and colleagues from Novartis to describe compounds in 
their high-throughput screening library which have failed to show biological activity in any screening assay, yet 
through gene-expression studies demonstrated the potential for biological activity in future screens. 
\cite{Wassermann2015}
These compounds offer interesting starting points for drug discovery as their lack of activity in historically 
target-driven screens may mean they have the potential to act through novel mechanisms of action.
A target agnostic approach coupled with unbiased detection of subtle biological activity positions high-content imaging 
as a useful tool to identify dark chemical matter in compound collections.
As I did not have access to historical records of the BioAscent's performance in a wide range of assays, I instead used 
the records in the ChEMBL database.
From the 1244 active BioAscent compounds, 59 where structurally distinct from any listed in the ChEMBL records (figure 
\ref{figure:dark_mols}).
There is also the possibility that there may be more dark chemical matter in the BioAscent library, as I did not 
investigate the bio-activity of the structurally similar records in the ChEMBL database, and that many of those which 
returned structural analogues may not have shown activity in previous assays.
As the BioAscent library has been designed around drug-like molecules, and a measure of drug-likeness did not reveal 
any undesirable physiochemical properties of these dark chemical matter the reason behind their exclusion in previous 
screening assays remains unclear.

Overall, incorporating cheminformatics and high-content screening presents an interesting opportunity for drug 
discovery by combining the well-defined and annotated cheminformatics field with the rich datasets high-content imaging 
can provide.
In this chapter I have shown that high-content screening data can be combined with existing datasets to aid 
interpretation using chemical structure as a common linker to retrieve data for either the same compound or similar 
compounds, as well as demonstrating the use of high-content screens to identify interesting areas of chemical space for 
the development of novel therapeutics.




\section{Methods}

\subsection{Chemical similarity}

Compound structural information was provided in the form of .sdf files by the supplier.
To create daylight-like compound fingerprints the RDKit library was used to convert .sdf entries into an RDKit's 
implementation of the daylight fingerprint using the `rdkit.Chem.Fingerprints.FingerprintMols' function with default 
parameters.

USRCAT features of the BioAscent library were generously calculated and supplied by Dr. Steven Shave (Edinburgh).

Latent representations of chemical structure features were calculated using a molecular autoencoder pre-trained on the 
ChEMBL22 dataset \footnote{www.github.com/cxhernandez/molencoder}, based on the work published by Gomez-Bombarelli 
\textit{et al.} \cite{Gomez-Bombarelli2016} using one-hot encoded SMILE strings of the molecules.

To compute the distance between RDKit daylight fingerprints the Tanimoto distance was used, in the case of USRCAT and 
autoencoded features I used the Euclidean distance.
Hierarchical clustering was performed on the distance matrix using the complete linkage method and euclidean distance.
To define clusters from the calculated dendrogram, a threshold was defined as 70\% of the maximum linkage distance.
Butina clustering was implemented using RDKit with Tanimoto distances calculated from daylight fingerprints, with a 
cutoff value of 0.2.

Mantel's test for comparing two distance matrices was implemented with scikit-bio's implementation using Pearson's 
correlation coefficient and 999 permutations for significance testing.
The distance matrices used were standardised Euclidean distance for the morphological profiles and standardised 
Tanimoto distances of the daylight fingerprints for compound structure profiles.


\subsection{BioAscent library screen}

The morphological data used in this chapter is from the MCF7 cell-line stained using the cell-painting protocol, imaged 
with the ImageXpress and morphological features calculated using Cellprofiler.

\subsubsection{Compound activity window}
Data was normalised to plate-based controls and features standardised, then transformed with PCA to the minimum number 
of principal components which accounted for 80\% of the variance in the data.
$l_1$ norm distances were calculated from the DMSO negative control centroid in PCA space.
The lower bound of the activity window was defined visually using a plot of ranked $l_1$ distances.
The upper bound was chosen based on images containing at least 10 cells and visual assessment of images produced by 
higher $l_1$ distances ensuring images did not consist entirely of dying cell (small, rounded and bright cytoplasmic 
staining).


\subsection{Phenotypic similarity}
Clustering of morphological profiles was carried out by first calculating a correlation matrix between between all 
pairs of active compound morphologies.
Hierarchical clustering was performed on the correlation matrix using the complete linkage method and euclidean 
distance.
To define clusters from the calculated dendrogram, a threshold was defined as 70\% of the maximum linkage distance 
which produced 10 clusters (figure \ref{figure:dendrogram_cut})

\begin{figure}
    \captionsetup{width=0.8\textwidth}
    \caption[Dendrogram threshold to determine clusters]{
Dendrogram thresholding to determine the number of phenotypic clusters in the active BioAscent compounds.
Dashed line indicates cutoff of 70\% of the maximum linkage distance, resulting in 10 clusters.
}
    \includegraphics[width=0.8\textwidth]{ch5dendrogram}
    \label{figure:dendrogram_cut}
\end{figure}

t-SNE clustering was performed using sklearn's `manifold.TSNE` implementation using the Barnes-Hut approximation with 
the default parameters.

\subsection{ChEMBL structure searches}
To programmatically query the ChEMBL database I used the python ChEMBL webresource client. 
\footnote{www.github.com/chembl/chembl\_webresource\_client}
In order to identify records for similar compounds I first queried structures based on SMILE strings of the BioAscent 
compounds with a filter to return only compounds with a Tanimoto similarity of 0.9, recording the similar compounds as 
ChEMBL identifiers.
Then in a second query using the ChEMBL identifiers, I searched for historical screening results against human protein 
targets and returned a list in the form of Uniprot accession codes.
As this returned a list of all protein targets which had been screened against, I filtered this list to protein targets 
with an assay EC/IC$_{50}$ value less than 1 $\mu$M.
This was repeated for each cluster of BioAscent compounds returning a list of Uniprot accession codes for each cluster.


\subsection{Dark chemical matter}
To search for active compounds in the BioAscent library which are structurally distinct from any compounds in the 
ChEMBL database I queried the ChEMBL webresource with the 1244 active BioAscent compounds, returning compounds within 
70\% similarity, which is equivalent of compounds within 0.3 Tanimoto distance (this is the minimum similarity value 
allowed when using ChEMBL's API).
Any BioAscent compound that failed to return any structurally similar ChEMBL record was listed as a `dark SMILE'.
\footnote{A thanks to Michał Nowotka from the EMBL-EBI for his help making changes to the ChEMBL servers and API to 
enable such time-intensive queries.}
QED values of drug-likeness were computed using RDKit Chem.QED.qed function with default parameters on molecules 
computed from the supplied sdf file.


\subsection{Interpro analysis}
Interpro analysis was carried out using DAVID 6.8. \cite{Huang2009}
DAVID was chosen despite more up-to-date alternatives, as DAVID allows uploading a custom background list of genes or 
proteins.
Therefore I created a background list of protein targets by repeating the Uniprot lookup as before but with a list of 
all 12,000 BioAscent compounds, which was used as a background for each cluster analysis with DAVID.
Significantly enriched interpro targets were selected based on a Benjamini-Hochberg corrected p-value with an $\alpha$ 
of 0.05.

\end{document}
